src/Voron/VoronConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Voron
+{
+    internal static class VoronConfiguration
+    {
+        // PERF: Because all those values are static readonly booleans that are defined during the process of loading the type,
+        // the JIT will detect them as such and will use the values instead. 
+        // https://alexandrnikitin.github.io/blog/jit-optimization-static-readonly-to-const/
+        
+        public static readonly bool FailFastForStability;
+
+        static VoronConfiguration()
+        {
+#if DEBUG
+            FailFastForStability = true;
+#else
+            if (Environment.GetEnvironmentVariable("RAVENDB_Voron_FailFast")?.ToLowerInvariant() == "true")
```

Comment:
Our env looks like RAVEN_HTTP_POOLEDCONNECTIONLIFETIMEINSEC, can you adjust? Can you fix as well the ones in AdvInstructionSet?

Reply:
bool.TryParse would be better? You do not have to deal with whitespaces then?